fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/plan/debug/Commands.java

```diff
@@ -310,7 +310,12 @@ public boolean executeCommand(@Nonnull final PlannerRepl plannerRepl,
                                       @Nonnull final Event event,
                                       @Nonnull final ParsedLine parsedLine) {
             final State state = plannerRepl.getCurrentState();
-            final Event e = state.getEvents().get(state.getCurrentTick());
+            final List<Event> events = state.getEvents();
```

Comment:
Event recording is switched off by default in `DebuggerWithSymbolTables` and switched on by default in `PlannerRepl`. In complex queries, the events array can take many MBs, so I wanted to reduce that foot print (I got into memory pressure trouble due to some bug I wasn't aware of at the time)